<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShortcutEditorDialog</class>
 <widget class="QDialog" name="ShortcutEditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>754</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Shortcut Configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QComboBox" name="objectsComboBox"/>
        </item>
        <item>
         <widget class="QTreeWidget" name="shortcutsTreeWidget"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QFrame" name="">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QGroupBox" name="groupBox_3">
              <property name="title">
               <string>Shortcuts</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_7">
               <item>
                <widget class="QComboBox" name="keyComboBox1"/>
               </item>
               <item>
                <widget class="QComboBox" name="keyComboBox2"/>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="modifiersGroupBox">
              <property name="title">
               <string>Modifiers</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <widget class="QCheckBox" name="ctrlCheckBox">
                 <property name="text">
                  <string>Ctrl</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="shiftCheckBox">
                 <property name="text">
                  <string>Shift</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="altCheckBox">
                 <property name="text">
                  <string>Alt</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="metaCheckBox">
                 <property name="text">
                  <string>Meta</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="autorepeatGroupBox">
              <property name="title">
               <string>Autorepeat</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Start Delay (ms)</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="startDelaySpinBox">
                 <property name="minimum">
                  <number>10</number>
                 </property>
                 <property name="maximum">
                  <number>500</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Interval (ms)</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="repeatIntervalSpinBox">
                 <property name="minimum">
                  <number>10</number>
                 </property>
                 <property name="maximum">
                  <number>100</number>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ShortcutEditorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ShortcutEditorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
